web-components/src/components/accordion/tokens/lm-accordion-tokens.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-undef */
+const colors = require("@momentum-ui/tokens/dist/colors.json");
+
+const accordion = {
+  prefix: "lm",
+  component: "accordion",
+  color: {
+    light: colors.gray[70].name,
+    dark: colors.gray[40].name
```

Comment:
this is available globally as `--md-secondary-text-color`

Reply:
@futurethang for now this styles is in draft status, because we don't have design spec for accordion. When design will be finalized we can completely finish styling